packages/next-on-pages/templates/_worker.js/utils/cache.ts

```diff
@@ -51,6 +54,9 @@ export async function handleSuspenseCacheRequest(request: Request) {
 			case 'POST': {
 				// Update the value in the cache.
 				const body = await request.json<IncrementalCacheValue>();
+				// Falling back to the cache tags header for Next.js 13.5+
+				body.tags = body.tags ?? body.data.tags ?? request.headers.get(CACHE_TAGS_HEADER)?.split(',') ?? [];
```

Comment:
I think we should also delete any `body.data.tags` property that exists so that we aren't storing the tags twice (on both `body.tags` and `body.data.tags`) for older Next.js versions.